src/data/species.yaml

```diff
@@ -53,9 +53,30 @@
     0a591d04-1904-466d-a198-e172a11a424e:23,
     dfca4764-06e7-4ee3-9ac8-aec65723bdec:100
   generatedId: fd6b29ee-aaf8-4a23-b8bd-990797ef915f
+- name: Dewback
+  player: No
```

Comment:
I _just_ deduplicated creatures from species today, so this column doesn't exist any more on species. Also, if any of these are creatures can you please make sure they are in creatures only and not also species?